### PR TITLE
add gnmi timestamp notes

### PR DESCRIPTION
**Pull request by sulrich**
- adds some documentation for notification timestamp modification
  - addresses #38 
- minor updates to formatting per style guide

### CODE REVIEW

File: docs/configuration/openconfig.md [hunk 1]
```diff
@@ -183,10 +182,50 @@ Starting in EOS `4.25.1F` it is possible to enable these mappings, for IPV4, IPV
...
+the send-time is available. (as of 4.27.0F)
+
+Overriding the timestamp to `send-time` is applicable to all STREAM and POLL subscriptions.
+
+Configuration is outlined below.
+
+```text
+management api gnmi
+   transport grpc default
+      notification timestamp send-time
```
Comment (noredistribution) [inline docs/configuration/openconfig.md:203]: should we mention that we can change it back to the defaults either by doing
`no notification timestamp send-time`
or `notification timestamp last-change-time` ?

**Review by noredistribution — COMMENTED** [review]
LGTM, left 1 comment

File: docs/configuration/openconfig.md (hunk 1 above)
Comment (sulrich) [reply docs/configuration/openconfig.md:203]: good idea.  i'll update.

**Comment by noredistribution** [comment]
LGTM +1

**Review by noredistribution — APPROVED** [review]

**Pull request merged**